src/rascal/representations/calculator_spherical_invariants.hh

```diff
@@ -426,8 +426,8 @@ namespace rascal {
 
         Key_t sparsified_type{0, 0};
         // there are 118 atomic elements atm
```

Comment:
Is this comment outdated? It reads like it is a leftover from when the loop was `for (int sp1{1}; sp1 < 118; sp1++)`